### PR TITLE
Port to BaseApp

### PR DESCRIPTION
**Pull request by tchx84**

### CODE REVIEW

**Comment by tchx84** [comment]
~~I am still in conversations with upstream regarding how to handle https://github.com/flathub/org.sugarlabs.Pippy/pull/1/files#diff-bca4a76f739209b3690f1c79c0b216a1~~

File: org.sugarlabs.Pippy.json [hunk 1]
```diff
@@ -242,6 +63,21 @@
...
+            "buildsystem" : "meson",
+            "build-options" : {
+                "env" : {
+                    "CXXFLAGS" : "-fno-exceptions"
+                }
+            },
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/vte.git"
```
Comment (bilelmoussaoui) [inline org.sugarlabs.Pippy.json:77]: I don't think it's a good idea to build vte from master

Comment (tchx84) [reply org.sugarlabs.Pippy.json:77]: Ouch~ I just merged~ it... if there's a way you can cancel it, I can fix this :)

Comment (bilelmoussaoui) [reply org.sugarlabs.Pippy.json:77]: You can stop the build from here https://flathub.org/builds/#/apps/org.sugarlabs.Pippy

Comment (tchx84) [reply org.sugarlabs.Pippy.json:77]: Done, thanks! anything else you can spot?

**Pull request merged**